config/initializers/devise.rb

```diff
@@ -0,0 +1,293 @@
+# frozen_string_literal: true
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'your_key_here'
+  config.secret_key = 'b0d7f10dab0977beed8c852a8da68f4d6de9045a1251216d915db23'\
```

Comment:
Wait, do we want this hardcoded? I feel like we want to just inherit from `secret_key_base` and make sure it's appropriately set when we run tests.